egs/swbd/s5c/local/chain/run_tdnn_attend1a.sh

```diff
@@ -119,11 +131,11 @@ if [ $stage -le 12 ]; then
 
   # the first splicing is moved before the lda layer, so no splicing here
   relu-batchnorm-layer name=tdnn1 dim=625
-  attention-relu-renorm-layer name=tdnn2 input=Append(-1,0,1) $attend_common
+  attention-relu-renorm-layer name=attention2 $attend_common
```

Comment:
How do you mean? I imagined we would need to try some different values for time-stride anyway, so this was a start.